### PR TITLE
artsem / Api playground layout part 1

### PR DESCRIPTION
**Pull request by artsemkavaleu-deriv**

### CODE REVIEW

File: src/components/common/TokenInputField/TokenInputField.module.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    border: none;
+    border-radius: 0 4px 4px 0;
+    background-color: #ff444f;
+    font-size: 14px;
+    line-height: 1.43;
+    color: #fff;
+
+    &:hover {
+        background-color: #d43e47;
+      }
```
Comment (maryia-deriv-archive) [inline src/components/common/TokenInputField/TokenInputField.module.scss]: please press option+shift+F in this file to prettify tabs 😊

File: src/styles/globals.scss [hunk 2]
```diff
@@ -86,4 +87,4 @@ h2 {
   line-height: 1.5;
```
Comment (maryia-deriv-archive) [inline src/styles/globals.scss]: it looks like option+shift+F is needed to prettify this file too ☺️

**Review by nikita-deriv — APPROVED** [review]
LGTM

File: src/components/common/TokenInputField/TokenInputField.module.scss (hunk 1 above)
Comment (artsemkavaleu-deriv) [reply src/components/common/TokenInputField/TokenInputField.module.scss]: as u wish, my mistress

File: src/styles/globals.scss (hunk 2 above)
Comment (artsemkavaleu-deriv) [reply src/styles/globals.scss]: as u wish, my mistress

**Pull request merged**